src/main/java/nl/knaw/dans/easy/sword2examples/ZipComponent.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.sword2examples;
+
+import java.io.IOException;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
```

Comment:
These classes handle less dialects of zip files than `org.apache.commons.compress.archivers.zip`
See https://github.com/DANS-KNAW/easy-deposit-api/pull/169

Reply:
I checked out two other libraries, compress and the zip4j, however neither gave me the filenames of the files contained in the zip directory in an easy to work with way. Because this library is now only used for checking the names of files, and not for extracting files I do not expect this to cause any problems.